runtime/services/multimediad/index.js

```diff
@@ -90,8 +90,8 @@ service.on('seekcomplete', function (id, dur, pos) {
     '/multimedia/service',
     'multimedia.service',
     'multimediadevent',
-    'ss',
-    [id, 'seekcomplete']
+    'ssss',
+    [id, 'seekcomplete', dur, pos]
```

Comment:
Also need update docs at https://github.com/yodaos-project/yodart/blob/master/runtime/lib/descriptor/multimedia-descriptor.js#L112-L119.

Reply:
BTW, why not using `ssdd` and the `d` stands for `double` in D-Bus.